components/admin_console/license_settings/enterprise_versions/enterprise_versions_right_panel.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import * as React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {ClientLicense} from 'mattermost-redux/types/config';
+
+import WomanUpArrowsAndCloudsSvg from 'components/common/svg_images_components/woman_up_arrows_and_clouds_svg';
+import ContactUsButton from 'components/announcement_bar/contact_sales/contact_us';
+import WomanWithCardSvg from 'components/common/svg_images_components/woman_with_card_svg';
+import TwoPeopleChattingSvg from 'components/common/svg_images_components/two_people_chatting_svg';
+
+export interface EnterpriseVersionsProps {
+    isTrialLicense: boolean;
+    license: ClientLicense;
+}
+
+const EnterpriseVersionsRightPanel: React.FC<EnterpriseVersionsProps> = ({
+    isTrialLicense,
+    license,
+
+}: EnterpriseVersionsProps) => {
+    const upgradeAdvantages = [
+        'AD/LDAP Group sync',
+        'High Availability',
+        'Advanced compliance',
+        'Advanced roles and permissions',
+    ];
+
+    const skuShortName = license.SkuShortName;
+
+    const contactSalesBtn = (
+        <div className='purchase-card'>
+            <ContactUsButton
+                eventID='post_trial_contact_sales'
+                customClass='light-blue-btn'
+            />
+        </div>
+    );
+
+    const title = () => {
+        if (isTrialLicense) {
+            return (
+                <FormattedMessage
+                    id='admin.license.purchaseEnterprisePlanTitle'
+                    defaultMessage='Purchase the Enterprise Plan'
+                />
+            );
+        }
+        if (skuShortName === 'enterprise') {
+            return (
+                <FormattedMessage
+                    id='admin.license.enterprisePlanTitle'
+                    defaultMessage='Need to increase your headcount?'
+                />
+            );
+        }
+        return (
+            <FormattedMessage
+                id='admin.license.upgradeToEnterprise'
+                defaultMessage='Upgrade to the Enterprise plan'
+            />
+        );
+    };
+
+    const svgImage = () => {
+        if (isTrialLicense) {
+            return (
+                <WomanWithCardSvg
+                    width={100}
+                    height={100}
```

Comment:
Are these mesuraments intended? It seems that they are still showing very small

Reply:
we are still checking the SVG not resizing problem.